src/main/java/io/split/android/client/service/sseclient/reactor/MySegmentsUpdateWorker.java

```diff
@@ -31,7 +31,7 @@ public MySegmentsUpdateWorker(
     protected void onWaitForNotificationLoop() throws InterruptedException {
         try {
             mNotificationsQueue.take();
-            mSynchronizer.synchronizeMySegments();
+            mSynchronizer.forceMySegmentsSync();
```

Comment:
check Javi, but `SynchronizerImpl::synchronizeMySegments` and `SynchronizerImpl::forceMySegmentsSync` seems to be doing the exact same thing. Consider removing one of them if that is the case, or probably `forceMySegmentsSync` requires to pass true for `avoidCache` flag.

Reply:
Good catch Doc. Fixed!